pkgs/by-name/ju/junest/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, wget, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  version = "7.4.8";
+  pname = "junest";
+  src = fetchFromGitHub {
+    owner = "fsquillace";
+    repo = "junest";
+    rev = "refs/tags/${version}";
+    hash = "sha256-9yrQ721fHUxXEZ0mh27SB8yoUH67ltOktUq3kr4qrBc=";
+  };
+
+  dontBuild = true;
+
+  buildInputs = [
+    makeWrapper
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    cp -r $src/bin/ $out/
+    cp -r $src/lib/ $out/
+    wrapProgram $out/bin/junest --prefix PATH ${wget}/bin
```

Comment:
wrapper still didnt work

```
cat ./results/junest/bin/junest 
#! /nix/store/dy2z01kpnxn7dn2kgfdxs4fm8xy9mb89-bash-5.2p26/bin/bash -e
exec -a "$0" "/nix/store/x4qb7mvhfi6xck6yyfqsi5b3r9d6q3s6-junest-7.4.8/bin/.junest-wrapped"  "$@" 
```

maybe you need `:` after PATH ?

Reply:
I built the package locally and can't find any problems. The wrapper generated works, and it seems to be using wget as intended. 

I ran:
```
nix build .#junest
```

Reply:
whats the content of 
```
cat ./results/junest/bin/junest 
```

my doesn't have wget PATH set

Reply:
After further investigation it is not in fact working. However, I have found a fix.

Reply:
Should work now!